### PR TITLE
#11374: Removed the announcement knowledge panel customisability

### PR DESCRIPTION
**Pull request by aniketkatkar97**
- [x] Added changes to remove announcement widget from customizable page and only show the widget on landing page in case there are announcements present

https://github.com/open-metadata/OpenMetadata/assets/51777795/30653471-94e5-47b7-bcf7-e41ea8e181eb

**updated styling for announcements:**

https://github.com/open-metadata/OpenMetadata/assets/51777795/a0a9efdb-4340-4fb8-bc1a-5bb18ce1d7ad

- [x] Added unit and cypress tests for Persona and Customizable landing page

I worked on ... because ...

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

**Comment by harshach** [comment]
@aniketkatkar97 lets finish up this and merge it in

**Review by chirag-madlani — APPROVED** [review]
Awesome job @aniketkatkar97 🚀 
Better to have smaller PRs next time for quick review.

**Comment by harshach** [comment]
@aniketkatkar97 is it possible to remove the border around announcements

File: openmetadata-ui/src/main/resources/ui/cypress/common/CustomizeLandingPageUtils.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    'GET',
+    `/api/v1/analytics/dataInsights/charts/aggregate*`,
+    'getDataInsightReport'
+  );
+
+  cy.get('#openmetadata_logo').click();
+
+  verifyResponseStatusCode('@getFeedsData', 200);
+  verifyResponseStatusCode('@getDataInsightReport', 200);
+};
```
[the remark this answers is not in the document]
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/cypress/common/CustomizeLandingPageUtils.js:99]: The command `gotToHomePage`  is written assuming the whats new modal will be visible so we use it just after login

**Review by harshach — APPROVED** [review]

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**